Source/IGListAdapter.m

```diff
@@ -414,6 +414,23 @@ - (NSArray *)visibleObjects {
     return [visibleObjects allObjects];
 }
 
+- (NSArray<UICollectionViewCell *> *)visibleCellsForObject:(id)item {
+    IGAssertMainThread();
+    IGParameterAssert(item != nil);
+    
+    NSArray<UICollectionViewCell *> *visibleCells = [self.collectionView visibleCells];
+    NSInteger section = [self.sectionMap sectionForObject:item];
+    IGAssert(section != NSNotFound, @"Section not found for object %@", item);
```

Comment:
Actually let's remove this. A use might be holding a ref to an expired object. I think it's ok to return an empty array in that case.